QUEST_LV_0100.tsv

```diff
@@ -3839,8 +3839,8 @@ QUEST_LV_0100_20150317_003838	Check status of the Scout
 QUEST_LV_0100_20150317_003839	Please check the status of the attack Scout to Kepa.
 QUEST_LV_0100_20150317_003840	Remind the Scout that there is an order to assemble to Klaipeda
 QUEST_LV_0100_20150317_003841	Remind the Scout again that there is an order to assemble to Klaipeda.
-QUEST_LV_0100_20150317_003842	Defeat the Leaf Bugs and collect the luggages.
-QUEST_LV_0100_20150317_003843	The Scout says he can't return until he finds his luggages. Find the Scout's stolen luggages from the Leaf Bugs.
+QUEST_LV_0100_20150317_003842	Defeat the Leaf Bugs at Uoros farm and collect the luggage.
```

Comment:
at Uoros farm is not mentioned in the tracker from what I saw, it's only highlighted on the map, but there is no label to it. I'll change that to belongings, then.